src/silo/query_engine/filter_expressions/symbol_equals.cpp

```diff
@@ -137,9 +133,7 @@ std::unique_ptr<silo::query_engine::operators::Operator> SymbolEquals<SymbolType
       );
       return std::make_unique<operators::Complement>(
          std::make_unique<operators::IndexScan>(
-            std::move(logical_equivalent_of_nested_index_scan),
```

Comment:
`logical_equivalent_of_nested_index_scan` is now unused.

Why is it ok that we don't pass that argument to `IndexScan` now? AFAICS this is debug output, but maybe valuable?

Reply:
Ahh, I temporarily removed this (due to cyclic dependency) but wanted to insert it back in, thanks for catching it!